database/postgres/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
+
+VERSION="1.24.0"
+NAMESPACE="postgres"
+
+# Install cloudnative-pg operator.
+kubectl apply --server-side -f \
+  https://raw.githubusercontent.com/cloudnative-pg/cloudnative-pg/main/releases/cnpg-${VERSION}.yaml
+kubectl rollout status deployment --namespace=cnpg-system --timeout=90s
+
+# Create the postgres database server.
+kubectl create ns ${NAMESPACE}
```

Comment:
Running `kubectl create ns` fails if the namespace already exists, use this other command to make the operation idempotent.

```suggestion
kubectl create ns ${NAMESPACE} --dry-run=client -o yaml | kubectl apply -f -
```

Reply:
Fixed.